server/controller/menu.js

```diff
@@ -54,6 +54,34 @@ class MenuController {
     });
     return null;
   }
+
+  /**
+  * UpdateFoodItem Controller updates a meal in the database
+  * @param {string} req - The request to the server
+  * @param {string} res - The response from the server.
+  */
+  static updateFoodItem(req, res) {
+    const { mealId } = parseInt(req.params, 10);
+    const {
+      mealName, mealImageUrl, mealDescription, mealPrice,
+    } = req.body;
+    const updateMealQuery = `UPDATE menu SET mealName = '${mealName}', mealImageUrl = '${mealImageUrl}', mealDescription = '${mealDescription}', mealPrice = '${mealPrice}' WHERE mealId = ${mealId} RETURNING *;`;
```

Comment:
#### Critical
Format this code to be more readable.

Reply:
I have implemented this feedback

Reply:
#### Nitpick
Put each of those variables on a new line since you're using template literals.